components/script/script_thread.rs

```diff
@@ -181,6 +185,8 @@ impl InProgressLoad {
             is_visible: true,
             url: url,
             origin: origin,
+            navigation_start: 0,
+            navigation_start_precise: 0.,
```

Comment:
To rephrase my question - why don't we get the current time here and store it immediately, instead of waiting for the constellation? This both simplifies the code and gives us a more accurate initial time.